cypress/integration/common/index.js

```diff
@@ -51,6 +51,21 @@ Given('user is on landing Genetics Terms page', () => {
 	cy.visit(baseURL);
 });
 
+
+Given('user enters {string} as the URL', (a) => {
+	cy.visit(a);
+});
+Then('the system redirects user to {string}',(a) =>{
+	
+	cy.location('pathname').should('eq','/def/a33');
```

Comment:
This step is also duplicate of existing redirect logic  - delete it

Reply:
ok